firebase-installations/src/main/java/com/google/firebase/installations/remote/RequestLimiter.java

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.installations.remote;
+
+import androidx.annotation.GuardedBy;
+import com.google.firebase.installations.Utils;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * The {@link RequestLimiter} class calculates the next allowed request time. Also, decides whether
+ * a network request to FIS servers is allowed to execute.
+ *
+ * @hide
+ */
+class RequestLimiter {
+  private static final long BACKOFF_TIME_24H_IN_MILLIS = TimeUnit.HOURS.toMillis(24);
+  private static final long BACKOFF_TIME_30_MINS_IN_MILLIS = TimeUnit.MINUTES.toMillis(30);
```

Comment:
These constant names currently only describe the actual value of the constant, but as reader I don't understand their business purpose.
It would be better, if the constant name describes the semantic meaning or what the constant is used for.

How about these names:
MAXIMUM_BACKOFF_INTERVAL_FOR_CONFIGURATION_ERRORS
MAXIMUM_BACKOFF_INTERVAL_FOR_ SERVER_ERRORS

Reply:
Maybe replace "...INTERVAL..." with  "...DURATION..."
I have just seen that you use the term "duration" below, instead of interval.
For some reason, I prefer "interval" today, haha, but
I think both are fine as long as we stay consistent.